MODULE.bazel

```diff
@@ -0,0 +1,17 @@
+module(
+    name = "rules_python",
+    version = "0.4.0",
```

Comment:
This is not the current release version.

Reply:
It perhaps makes sense to set this to 0.0.0 and automatically commit from CI when doing a release?

Reply:
yes! actually 0.0.0 can stay here, the file in the registry is modified